components/Filter/FilterAsset.tsx

```diff
@@ -80,133 +60,26 @@ const FilterAsset: NextPage<Props> = ({
     { fallback: 'sm' },
   )
 
+  const formValues = useForm<Filter>({
+    defaultValues: filter,
+  })
   const {
-    register,
-    formState: { isSubmitting, errors },
+    formState: { isSubmitting },
     handleSubmit,
     reset,
-    setValue,
-    control,
     watch,
-  } = useForm<Filter>({
-    defaultValues: filter,
-  })
+  } = formValues
   const filterResult = watch()
+
   useEffect(() => {
     reset(filter)
     return () => reset(NoFilter)
   }, [reset, filter])
 
-  const { data: currencyData } = useChainCurrenciesQuery({
-    variables: {
-      filter:
-        filterResult.chains.length > 0
-          ? ({ chainId: { in: filterResult.chains } } as CurrencyFilter)
-          : (undefined as unknown as InputMaybe<CurrencyFilter>),
-    },
-    ssr: false,
-  })
-  const currencies = useMemo(
-    () => currencyData?.currencies?.nodes || [],
-    [currencyData],
-  )
-
-  const currency = useMemo(
-    () =>
-      currencies.length === 1
-        ? currencies[0]
-        : currencies.find((x) => x.id === filterResult.currency?.id) ||
-          currencies[0],
-    [currencies, filterResult.currency],
-  )
-
-  const [collectionSearch, setCollectionSearch] = useState('')
+  const [collection, setCollection] = useState<
+    { chainId: number; address: string } | undefined
+  >(undefined)
```

Comment:
Too fix the empty traits filter on the collection page, maybe setting the default state to `selectedCollection` will solve the issue:
```suggestion
  >(selectedCollection)
```

Reply:
Also the param `selectedCollection` has a very confusing name. This is not the collection selected by the user, but the collection we want to force.
Maybe renaming it to `forceCollection` will be better?

Reply:
Changed to `currentCollection` instead of `selectedCollection`. We only use that prop on collection page anyway.
[Change naming](https://github.com/liteflow-labs/starter-kit/pull/375/commits/4870966391c21449b25cf78198f305252087b8f5)